src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = (
+  { tabs, selectedTabId, onTabSelected },
+) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const handleTab = (tab: Tab) => {
+    if (selectedTabId !== tab.id) {
+      onTabSelected(tab);
+    }
+  };
```

Comment:
git rid of this function, just move if statement into `handleSelectTab`  and you could use `onTabSelected` instead of `handleTab `

Reply:
Fixed, but had to put this statement inside a`s onClick 
if (id !== selectedTabId) {
  onTabSelected(tab);
}
to prevent invoking onTabSelected if the current tab is clicked